src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkDemandIndex/index.tsx

```diff
@@ -0,0 +1,135 @@
+import {
+  Clickable,
+  Flex,
+  InfoCircleIcon,
+  Spacer,
+  Text,
+  Tooltip,
+} from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Media } from "Utils/Responsive"
+import { MyCollectionArtworkDemandIndex_marketPriceInsights } from "__generated__/MyCollectionArtworkDemandIndex_marketPriceInsights.graphql"
+import { DemandIndexBar } from "./DemandIndexBar"
+
+interface MyCollectionArtworkDemandIndexProps {
+  marketPriceInsights: MyCollectionArtworkDemandIndex_marketPriceInsights
+}
+
+const DemandIndexExplanation =
+  "Overall strength of demand for this artist and medium combination. Based on the last 36 months of auction sale data from top commercial auction houses."
+
+const MyCollectionArtworkDemandIndex: React.FC<MyCollectionArtworkDemandIndexProps> = ({
+  marketPriceInsights,
+}) => {
+  if (!marketPriceInsights?.demandRank) {
+    return null
+  }
+
+  const demandRank = Number((marketPriceInsights.demandRank * 10).toFixed(2))
+
+  const adjustedDemandRank =
+    demandRank.toFixed(1) === "10.0" ? "9.9" : demandRank
+
+  const isHighDemand = Number(demandRank) >= 9
+  const demandRankColor = demandRank >= 7 ? "blue100" : "black60"
+
+  let barWidth = demandRank * 10
+  if (barWidth > 100) {
+    barWidth = 100
+  }
```

Comment:
(non-blocking): This might be easier to understand when reading the code

```suggestion
  const barWidth = Math.min(demandRank * 10, 100)
```